pkgs/tools/misc/ramfetch/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchgit
+}:
+
+stdenv.mkDerivation rec {
+  name = "ramfetch";
+  version = "1.1.0";
+
+  src = fetchgit {
+    url = "https://codeberg.org/o69mar/ramfetch.git";
+    rev = "v${version}";
+    hash = "sha256-XUph+rTbw5LXWRq+OSKl0EjFac+MQAx3NBu4rWdWR3w=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/bin"
+    install -Dm755 ramfetch $out/bin/ramfetch
```

```suggestion
    install -D ramfetch $out/bin/ramfetch
```

-m 755 is the default